components/ChangePercent/ChangePercent.tsx

```diff
@@ -14,21 +15,31 @@ type ChangePercentProps = {
 };
 
 export const ChangePercent: FC<ChangePercentProps> = ({ value, decimals = 2, ...rest }) => {
-	const isPositive = wei(value ?? 0).gt(0);
+	const isValid = value;
```

Comment:
Maybe this can be:
```suggestion
	const isValid = !!value;
```
to ensure the value is an actual boolean.